app/controllers/admin/edition_translations_controller.rb

```diff
@@ -54,8 +54,12 @@ def translated_item_name
   end
 
   def load_translated_models
-    @document_remarks = Document::PaginatedRemarks.new(@edition.document, params[:remarks_page])
-    @document_history = Document::PaginatedHistory.new(@edition.document, params[:page])
+    if preview_design_system?(next_release: false)
+      @document_history = Document::PaginatedTimeline.new(document: @edition.document, page: params[:page] || 1)
```

Comment:
**Minor naming opinion:**

Would it make more sense to call this `@document_timeline`?

Reply:
**Counter argument:**

It gets rendered inside the "History" tab, by the "Document History Tab" component.

So I guess `@document_history` fits in with that naming convention!